<?xml version="1.0" ?>

<launch>

<node pkg="homework2" type="homework2_pubsub" name="pubsub_node" />

<node pkg="homework2" type="homework2_advertiser" name="service_advertiser_node" /> 

<node pkg="homework2" type="test_homework2" name="test_homework2" output="screen" />

</launch>
